config/resolvers/config-feature-flags.yaml

```diff
@@ -23,18 +23,10 @@ metadata:
     app.kubernetes.io/part-of: tekton-pipelines
 data:
   # Setting this flag to "true" enables remote resolution of Tekton OCI bundles.
```

Comment:
I guess the comment sounds a bit strange now that the default is "true".

Reply:
Good catch!

Reply:
Waiting on acting on this until it's confirmed that the switch to the built-in resolvers being on by default should be moved out of this PR.

Reply:
let's keep the one PR